pycbc/waveform/parameters.py

```diff
@@ -205,39 +205,39 @@ def docstr(self, prefix='', include_label=True):
 q = Parameter("q",
                 dtype=float, label=r"$q$",
                 description="The mass ratio, m1/m2, where m1 >= m2.")
-m_p = Parameter("m_p",
-                dtype=float, label=r"$m_{\mathrm{pr}}$",
+primary_mass = Parameter("primary_mass",
```

Comment:
@ahnitz Yeah, it still means the primary mass. I agree, it's a bit confusing, but that seems to be the convention that's been adopted by the collaboration. As you say, it only affects the labeling though, so can easily be overridden.